### ISSUE TITLE
Unknown provider: agcLibraryLoaderProvider

### ISSUE THREAD

**Issue by Shane325** [angular-google-chart/angular-google-chart#336]
I'm getting the following error when I try and inject `agcLibraryLoaderProvider` as a dependency:

```
Error: [$injector:unpr] Unknown provider: agcLibraryLoaderProvider
http://errors.angularjs.org/1.5.9/$injector/unpr?p0=agcLibraryLoaderProvider
```

I am following the example from the documentation [here](http://angular-google-chart.github.io/angular-google-chart/docs/1.0.0-beta.1/providers/agcLibraryLoaderProvider/)

Here is how my code looks:

```
(function () {
  let myApplication = angular.module('app.abc', ngModules)

  fetchData().then(bootstrapApplication)

  function fetchData () {
    let initInjector = angular.injector([ 'ng' ])
    let $http = initInjector.get('$http')
    let $q = initInjector.get('$q')
    let userData = {}
    let billingData = {}
    let promises = [
      $http.get('/auth/user'),
      $http.post('/billing/plans')
    ]
    return $q.all(promises)
      .then((responses) => {
        userData = responses[ 0 ].data
        billingData = responses[ 1 ].data
      })
      .catch((res) => {
        console.log(`Failed to resolve initial promises: ${JSON.stringify(res)}`)
        if (Raven.isSetup()) {
          Raven.captureException(`Failed to resolve initial promises: ${JSON.stringify(res)}`)
        }
      })
      .finally(() => {
        myApplication
          .constant('USER_DATA', userData)
          .constant('BILLING_DATA', billingData)
          .config(config)
          .config(routing)
          .config(theming)
          .config(icons)
          .config(googleChartLibraryLoader)
          .config(googleChartGstaticLoader)
          .run(stateChange)
          .controller('AppController', AppController)
      })
  }

  function bootstrapApplication () {
    angular.element(document).ready(function () {
      try {
        angular.bootstrap(document, [ 'app.abc' ])
      } catch (err) {
        console.log(err)
        if (Raven.isSetup()) {
          Raven.captureException(`Failed to start abc app: ${err}`)
        }
      }
    })
  }

  googleChartLibraryLoader.$inject = ['agcLibraryLoaderProvider']

  function googleChartLibraryLoader (agcLibraryLoaderProvider) {
    // Select the loader strategy.
    agcLibraryLoaderProvider.setLoader('gstatic')
  }

  googleChartGstaticLoader.$inject = ['agcGstaticLoaderProvider']

  function googleChartGstaticLoader (agcGstaticLoaderProvider) {
    // Provider supports method chaining.
    agcGstaticLoaderProvider
      .setVersion('45')
      .addPackage('map')
      .setOption('mapsApiKey', 'MY_API_KEY')
  }
}())
```

I have a GeoChart that uses markers and I need to set a google API key to get it to work. Hence the need for the above providers. 

Any ideas?

**Comment by nbering** [comment]
First, be sure you're using v1.0.0-beta.1. It's the first and only version with support for gstatic. I'll try and do some work to take the beta label off it soon. I usually just label it beta to keep people from auto-updating until the community has a chance to report bugs.

**Comment by nbering** [comment]
Also... is the module name 'googlechart' included in the variable `ngModules`?

**Comment by Shane325** [comment]
Yes

```
let ngModules = [
  // angular dependencies
  angularMaterial, ngMessages, ngAnimate, ngMaterialDataTable, ngSanitize, ngCookies,
  // third party dependencies
  uirouter, 'googlechart', 'ngclipboard', 'ngMask'
]
```

I am also using v1.0.0-beta.1

**Comment by Shane325** [comment]
The issue was with my webpack configuration. It is fixed now,

**Comment by nbering** [comment]
I just wanted to add this link for historical purposes:

[Gitter Channel Archive - July 18, 2018](https://gitter.im/angular-google-chart/angular-google-chart/archives/2018/07/18)

In the chat, I had noticed that the NPM distribution of v1.0.0-beta.1 does not match the CDN distribution, for the minified version of the library.

I've added a release note to the GitHub releases tag to indicate the discrepancy. I also did a little work this week to clean up the bit-rot in the tests (Karma wasn't working because Karma has changed and package.json referenced "latest"). If I find the time, I'll try to get a beta.2 release out with a working minified bundle to NPM. I may also switch to the gstatic loader as the default.
